Assets/SEE/Game/MoveGizmo.cs

```diff
@@ -0,0 +1,109 @@
+﻿using UnityEngine;
+
+namespace SEE.UI3D
+{
+    /// <summary>
+    /// This gizmo represents the movement of elements of a city visually.
+    /// </summary>
+    internal class MoveGizmo : MonoBehaviour
+    {
+        /// <summary>
+        /// The start position of the movement visualization.
+        /// </summary>
+        private Vector3 start;
+
+        /// <summary>
+        /// The end position of the movement visualization.
+        /// </summary>
+        private Vector3 end;
+
+        /// <summary>
+        /// The material of the filled rectangle.
+        /// </summary>
+        private Material fillRectangleMaterial;
+
+        /// <summary>
+        /// The material for the outlined rectangle.
+        /// </summary>
+        private Material outlineRectangleMaterial;
+
+        /// <summary>
+        /// The material of the line between the start- and end-position.
+        /// </summary>
+        private Material directLineMaterial;
+
+        private static readonly int colorProperty = Shader.PropertyToID("_Color");
+
+        /// <summary>
+        /// Creates a new move-gizmo.
+        /// </summary>
+        /// <returns>The gizmo.</returns>
+        internal static MoveGizmo Create()
+        {
+            GameObject go = new("MovePivot");
+            MoveGizmo p = go.AddComponent<MoveGizmo>();
```

Comment:
We usually discourage one-letter variable names like these in our style guidelines.